reclass/config.py

```diff
@@ -36,6 +36,9 @@ def make_db_options_group(parser, defaults={}):
     ret.add_option('-z', '--ignore-class-notfound', dest='ignore_class_notfound',
                    default=defaults.get('ignore_class_notfound', OPT_IGNORE_CLASS_NOTFOUND),
                    help='decision for not found classes [%default]')
+    ret.add_option('-a', '--add-subdir-to-node', dest='compose_node_name', action="store_true",
```

Comment:
'-a', '--add-subdir-to-node' 
to
'-a', '--compose-node-name'

Reply:
thanks for spotting that. Fixed